reporting/report_carpentry-uio_2019.md

```diff
@@ -1,29 +1,59 @@
-Carpentry@UiO Annual Report - 2019
+# Carpentry@UiO 2019 Annual Report
 
-# Activities in the Spring Term
-- See report for spring term 2019 [Report Spring 2019](https://github.com/uio-carpentry/organisational/blob/master/reporting/summary_feedback_carpentries_spring_2019.md)
-# Activities in the Fall Term
-- Create report based on spring term 2019
+## Activities in the Spring Term 2019
+
+- See the detailed report for the spring term 2019 [Report Spring 2019](https://github.com/uio-carpentry/organisational/blob/master/reporting/summary_feedback_carpentries_spring_2019.md)
+
+## Activities in the Fall Term 2019
+
+- TO-DO: Create report based on spring term 2019
 - include separate evaluation forms for Databases and SQL and GitHub
-## Workshops
-- GitHub
-- Plotting and Programming with Python
-- Databases and SQL
-- R for Reproducible Research
+
+### Workshops
+
+- Programming with Python (novices) - Feb 8, 2019
+- Introduction to Version Control with Git (novices) - Feb 15, 2019
+- Git in Practice (intermediate) - Feb 27, 2019
+- Data Analysis and Visualisation in R for Ecologists - Mar 8, 2019
+- Unix Shell (novices) - Apr 24, 2019
+- Data Analysis and Visualisation in R for Ecologists - May 15, 2019
+- Introduction to R with Tidyverse (pt. 1) - May 22, 2019
+- Introduction to R with Tidyverse (pt. 2) - May 29, 2019
+- Getting your hands on climate data (novices) - Jun 7, 2019
+- Writing and Publishing on the Web Together Using GitHub - Sep 11, 2019
+- Plotting and Programming with Python - Sep 16, 2019
+- Databases and SQL - Oct 28, 2019
+- R for Reproducible Scientific Analysis - Nov 19, 2019
+
+#### Carpentry workshops during Research Bazaar 2019
+
+- Learn Deep-Learning with Python (2 days)
+- Get (even) smarter with the Unix Shell
+- Publication Ready Scientific Reports with Jupyter Notebook (1/2 day)
+- Getting Started with Open Science Framework (1/2 day)
+
 ## Community Building & Social Events
-- Christmas Party Dec 12
-- Onboarding sessions on Sep 2
-- coordination w/ NT (CodeRefinery/Carpentries Hub) and w/ MG (Digital Scholarship Centre)
-## StudyGroup
+
+- two onboarding sessions for new instructors and helpers were held on Sep 2
+- a Call for Helpers was sent to former workshop participants, new helpers and instructors were recruited (see [## Recruiting])
+- coordination with the newly appointed Regional Coordinator for the Nordic and Baltic Countries was established, with Oslo becoming a CodeRefinery and Carpentries Hub
+- coordination with the Digital Scholarship Centre (University Library) was established to serve as a virtual "home" for Carpentry@UiO workshops and activities
```

Comment:
We don't have a Digital Scholarship Centre yet at UiO but there is a process to establish one. Maybe rephrase a little? And explain what it is (going to be)?

Reply:
Thanks! This really is a draft, but I needed to give NT access to it :)